backend/app/views/backoffice/project_developers/_section_profile.html.erb

```diff
@@ -0,0 +1,14 @@
+<div>
+    <label class="flex items-center gap-3">
+      <%= t("backoffice.common.view_content_in") %>
+      <%= select_tag :content_lang, options_from_collection_for_select(Language.all, "id", "name", content_language),
+         class: "select",
+         onchange: "
```

Comment:
I was lazy to add stimulus here ;]

Reply:
feels a bit like something which will be part of every translatable form. Maybe I would move this to `base` folder and added it as partial to `form`.

Reply:
True, I move around files when adding a next form for investor.